distribution/lib/Standard/Base/0.0.0-dev/src/Main.enso

```diff
@@ -94,10 +94,6 @@ import project.Data.Text.Line_Ending_Style.Line_Ending_Style
 import project.Data.Text.Location.Location
 import project.Data.Text.Matching_Mode.Matching_Mode
 import project.Data.Text.Regex
```

Comment:
Should include Regex_Option here I think as well.

Reply:
Actually I meant to remove that, since nothing is using it.